### ISSUE TITLE
Missing windows installed from latest Release package 

### ISSUE THREAD

**Issue by toddaustin07** [SmartThingsCommunity/smartthings-cli#460]
**What is your question? Please describe.**
Can you please add a .msi file? :-)

**Additional context**
Add any other context or screenshots about the question here.

**Comment by keithnicholson** [comment]
Yes, please provide the MSI to the Windows x64 zip package.  It contains the executable.  If it takes a moment through out dev and ci/cd, can you let us know what the MSI does?  I suspect it installs to program files and sets an environment variable.

**Comment by albertovm** [comment]
I can use only in MacOS... not available for Windows 64.
